changed require for the socks5-http-client and socks5-https-client

david1983 opened the pull request:
Hi,
I noticed that the require for the socks5-http-request library was pointing on a wrong path when the library was inside the node_modules directory.

Quick fix and thank you for the library.

Commit:
changed require for the socks5-http-client

cuzzo commented:
Hey @david1983 ,

I made a similar fix, which I think is a little bit simpler. Can you let me know if that works for you?

Thanks!

snyk-bot committed:
fix: .snyk & package.json to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/npm:lodash:20180130

david1983 committed:
Merge pull request #1 from david1983/snyk-fix-0ok90s

[Snyk] Fix for 1 vulnerable dependencies

atd-schubert commented on index.js:
```diff
@@ -1,9 +1,9 @@
-var Socks5ClientHttpAgent = require('./node_modules/socks5-http-client/lib/Agent');
-var Socks5ClientHttpsAgent = require('./node_modules/socks5-https-client/lib/Agent');
+var path = require('path');
+var Socks5ClientHttpAgent = require(path.dirname(require.main.filename) + '/node_modules/socks5-http-client/lib/Agent');
```
I think the [other PR](https://github.com/atd-schubert/torrequest/pull/3/files) solves this problem even better. I don't know synk, but it seems to be similar to the audit functionality of npm, right?

atd-schubert reviewed (commented):
Sorry for the very big delay on this module. I haven't noticed the PRs.

When I compare the two PRs, I like the other solution more then this one with the path module.

I haven't used Snyk before and asking myself if it is better to use npm audit for that.

We can start a discussion about that here...
